### DISCUSSION TITLE
期中考問題

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1706] [General]
請問有說期中考是哪一天嗎，我看課程簡介寫是11/22號，但看起來今天沒有要考的意思(?，害我有點困惑

**Comment by chia-kai-chang** [comment]
各位同學好，
 
期中考日期為11/29 13:00~16:00

現場筆試，請攜帶2B鉛筆來至希館202教室參加期中考試
 
填寫完作答試卷後，請繳回試卷與答案卡，並請填寫問卷。
 
考題類似於NCUEDU的Quiz GPT：https://ncuedu.tw/learning/chatgpt/cid/1/qid/0 ，都是四選一的單選選擇題

請踴躍把握~ 

謝謝大家
